dowhy/causal_estimator.py

```diff
@@ -4,13 +4,22 @@
 import pandas as pd
 import sympy as sp
 
+from sklearn.utils import resample
 
 class CausalEstimator:
     """Base class for an estimator of causal effect.
 
     Subclasses implement different estimation methods. All estimation methods are in the package "dowhy.causal_estimators"
 
     """
+    # The default number of simulations for statistical testing
+    DEFAULT_NUMBER_OF_SIMULATIONS_STAT_TEST = 1000
+    # The default number of simulations to obtain confidence intervals
+    DEFAULT_NUMBER_OF_SIMULATIONS_CI = 100
+    # The portion of the total size that should be taken each time to find the confidence interval
+    # 1 is the recommended value 
+    # https://ocw.mit.edu/courses/mathematics/18-05-introduction-to-probability-and-statistics-spring-2014/readings/MIT18_05S14_Reading24.pdf
+    DEFAULT_SAMPLE_SIZE = 1
```

Comment:
Naming of variable: this is not really sample size, but rather the multiplier to the given sample size. how about sample_size_multiplier or some other name? Given that the multiplier will be fraction between (0,1], how about "sample_size_fraction"? that's closer semantically.